src/components/Camera/Liveness/Challenge.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react'
+import { h } from 'preact'
+import classNames from 'classnames'
+import style from './style.css'
```

Comment:
we are getting quite deep in our nesting, the style selector generator will not namespace these folders, we are risking collision quite a bit.